packages/nimble-blazor/NimbleBlazor/Components/NimbleDialog.razor.cs

```diff
@@ -17,6 +17,12 @@ public partial class NimbleDialog<TCloseReason> : ComponentBase
     [Parameter]
     public bool? PreventDismiss { get; set; }
 
+    [Parameter]
```

Comment:
Can we go ahead and add the typical public docs for this class?

Reply:
Also, we should probably also start writing the boiler-plate tests for each property we add (tests that make sure we update the DOM as expected for each public property).